plugins/render/api_key_test.go

```diff
@@ -0,0 +1,37 @@
+package render
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.APIKey: "rnd_Z7xMKp4NX1FoQNRyBpZs9yxDbu3i",
+			},
```

Comment:
Let's please add the expected output to this test case.

Reply:
> Let's please add the expected output to this test case.

I have updated it. Tests keep failing and the error message isn't clear enough. I'm not really conversant with golang.

Reply:
nevermind, i have resolved it.